Build broken due to missing package javax.annotation on Java 11

`mvn clean test` fails to build on Java 11 due to:

```
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ mariaDB4j-core ---                                                 
[INFO] Changes detected - recompiling the module!                                                                                     
[INFO] Compiling 9 source files to /var/home/vorburger/git/github.com/vorburger/MariaDB4j/mariaDB4j-core/target/classes               
[INFO] -------------------------------------------------------------                                                                  
[ERROR] COMPILATION ERROR :                                                                                                           
[INFO] -------------------------------------------------------------                                                                  
[ERROR] /var/home/vorburger/git/github.com/vorburger/MariaDB4j/mariaDB4j-core/src/main/java/ch/vorburger/mariadb4j/MariaDB4jService.ja
va:[29,24] package javax.annotation does not exist                                                                                    
[ERROR] /var/home/vorburger/git/github.com/vorburger/MariaDB4j/mariaDB4j-core/src/main/java/ch/vorburger/mariadb4j/MariaDB4jService.ja
va:[30,24] package javax.annotation does not exist                                                                                    
[ERROR] /var/home/vorburger/git/github.com/vorburger/MariaDB4j/mariaDB4j-core/src/main/java/ch/vorburger/mariadb4j/MariaDB4jService.java:[63,6] cannot find symbol
  symbol:   class PostConstruct
  location: class ch.vorburger.mariadb4j.MariaDB4jService
[ERROR] /var/home/vorburger/git/github.com/vorburger/MariaDB4j/mariaDB4j-core/src/main/java/ch/vorburger/mariadb4j/MariaDB4jService.ja
va:[79,6] cannot find symbol
  symbol:   class PreDestroy
  location: class ch.vorburger.mariadb4j.MariaDB4jService
[INFO] 4 errors 
```

This is because `javax.annotation` needs to be added as an explicit dependency to be able to build on Java 11 - I'll try to fix this.

This blocks #334.

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,6 +15,7 @@ jdk:
 #  - .mvn
 
 before_install:
-  - mvn -f DBs/pom.xml clean install
+# TODO uncomment this when raising a PR to fix https://github.com/vorburger/MariaDB4j/issues/484
+#  - mvn -f DBs/pom.xml clean install
 
 script: mvn package -B -V

--- a/mariaDB4j-core/pom.xml
+++ b/mariaDB4j-core/pom.xml
@@ -12,6 +12,10 @@
 
     <dependencies>
         <dependency>
+        	<groupId>jakarta.annotation</groupId>
+        	<artifactId>jakarta.annotation-api</artifactId>
+      	</dependency>
+        <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>

--- a/mariaDB4j-core/src/main/java/ch/vorburger/mariadb4j/DBConfigurationBuilder.java
+++ b/mariaDB4j-core/src/main/java/ch/vorburger/mariadb4j/DBConfigurationBuilder.java
@@ -252,11 +252,11 @@ public class DBConfigurationBuilder {
         String databaseVersion = getDatabaseVersion();
         if (databaseVersion == null) {
             if (OSX.equals(getOS()))
-                databaseVersion = "mariadb-10.3.16";
+                databaseVersion = "mariadb-10.2.11";
             else if (LINUX.equals(getOS()))
-                databaseVersion = "mariadb-10.3.16";
+                databaseVersion = "mariadb-10.2.11";
             else if (WIN32.equals(getOS()))
-                databaseVersion = "mariadb-10.3.16";
+                databaseVersion = "mariadb-10.2.11";
             else
                 throw new IllegalStateException(
                         "OS not directly supported, please use setDatabaseVersion() to set the name "

--- a/mariaDB4j/pom.xml
+++ b/mariaDB4j/pom.xml
@@ -21,17 +21,17 @@
 		<dependency>
 			<groupId>ch.vorburger.mariaDB4j</groupId>
 			<artifactId>mariaDB4j-db-linux64</artifactId>
-			<version>10.3.16</version>
+			<version>10.2.11</version>
 		</dependency>
 		<dependency>
 			<groupId>ch.vorburger.mariaDB4j</groupId>
 			<artifactId>mariaDB4j-db-win32</artifactId>
-			<version>10.3.16</version>
+			<version>10.2.11</version>
 		</dependency>
 		<dependency>
 			<groupId>ch.vorburger.mariaDB4j</groupId>
 			<artifactId>mariaDB4j-db-mac64</artifactId>
-			<version>10.3.16</version>
+			<version>10.2.11</version>
 		</dependency>
 		<dependency>
 			<!-- For MariaDB4jSpringService, only. This has to be repeated from mariaDB4j-core here due to the optional true below -->

